bip-musig2.mediawiki

```diff
@@ -225,30 +221,30 @@ The KeyGen Context is a data structure consisting of the following elements:
 
 We write "Let ''(Q, gacc, tacc) = keygen_ctx''" to assign names to the elements of a KeyGen Context.
 
-'''''GetXonlyPubkey(keygen_ctx)''''':
+Algorithm '''GetXonlyPubkey(keygen_ctx)''':
```

Comment:
```suggestion
Algorithm ''GetXonlyPubkey(keygen_ctx)'':
```
I think that looks even better and the italics match the font when we call the algorithms. But it's matter of taste in the end.

Reply:
Ok, done.